src/styles/_catalog.scss

```
@@ -1,22 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-height);
+  gap: 48px;
+  justify-content: center;
```

You don't need variable $card-height here, you should create variable $card-width and use it for creating grid columns. 
```suggestion
  display: grid;
  grid-template-columns: $card-width);
  gap: 48px;
  justify-content: center;
```

---

**FIxed**